woocommerce/woocommerce
# Remove Blocks Plugin Notif

### All Submissions:


### Changes proposed in this Pull Request:

Per discussion with @mikejolley - this branch seeks to remove the code used to generate the admin alert that encourages users to install the Blocks feature plugin. Since this code will be part of core as of 3.6 - the alert is no longer needed.

![block-notif](https://user-images.githubusercontent.com/22080/54049939-d2d95580-4192-11e9-9713-808efa87b88c.png)

### How to test the changes in this Pull Request:

1. On a site running 4.9.x, install Woo, and the Gutenberg plugin to see the notice
2. Apply this branch, and verify the notice is no longer shown.

### Other information:




### Changelog entry

Removing admin alert for the WooCommerce Gutenberg Products Block feature plugin.
